metabase/metabase
# Migrate admin to the new database form

Epic https://github.com/metabase/metabase/issues/26178

How to test:
- Go to admin -> databases
- Make sure you can add and edit a database via the form

Note: I still need to cleanup database types and I'll do that in the next PR.

<img width="671" alt="Screenshot 2022-12-02 at 18 32 39" src="https://user-images.githubusercontent.com/8542534/205340368-942d7729-5d79-4338-94f2-a7386acb474d.png">
